### PR TITLE
Upgrade to Gradle 8.10.2

### PR DESCRIPTION
**Pull request by Mikexec**

### CODE REVIEW

File: CHANGELOG.md
Comment (Natchica) [inline CHANGELOG.md:1]: The CHANGELOG modification need to be written at the bottom of the part it belongs to

**Comment by Natchica** [comment]
You could rename the PR to have a nicer name than just the branch name

File: CHANGELOG.md
Comment (jbern0rd) [reply CHANGELOG.md:1]: I agree with this

**Comment by jbern0rd** [comment]
> You could rename the PR to have a nicer name than just the branch name

This should have happened because the branch already had several commits when the PR was created.

**Review by jbern0rd — APPROVED** [review]

**Pull request merged**